.github/workflows/ci_workflow.yml

```diff
@@ -347,7 +347,7 @@ jobs:
             python-version: '3.11'
           # Keeping one universal build
           - build: 4
-            arch-type: "x86_64;arm64"
+            arch-type: "x86_64"
```

Comment:
Please revert this line, we want to keep at least one universal build.

Reply:
> Please revert this line, we want to keep at least one universal build.

Will do